package/pom.xml

```diff
@@ -101,11 +101,29 @@
               <relocations>
                 <relocation>
                   <pattern>com.google.protobuf</pattern>
-                  <shadedPattern>io.glutenproject.shaded.com.google.protobuf</shadedPattern>
+                  <shadedPattern>${gluten.shade.packageName}.com.google.protobuf</shadedPattern>
+                </relocation>
+                <!-- shaded com.google.guava -->
+                <relocation>
+                  <pattern>com.google.common</pattern>
+                  <shadedPattern>${gluten.shade.packageName}.com.google.common</shadedPattern>
+                  <includes>
+                    <include>com.google.common.**</include>
+                  </includes>
+                  <excludes>
+                    <exclude>com.google.common.jimfs.**</exclude>
+                  </excludes>
+                </relocation>
+                <relocation>
+                  <pattern>com.google.thirdparty</pattern>
```

Comment:
Can we also shade `com.google.j2objc` and `com.google.gson` ? I saw the Spark connect Server module shade these things and I think it's reasonable.

Reply:
Sure, i'll do rebase and shade those.